l10n_it_reverse_charge/tests/test_rc.py

```diff
@@ -36,6 +36,12 @@ def setUp(self):
             'supplier': True,
             'property_account_position_id': self.fiscal_position_intra.id
         })
+        self.supplier_intraEU_esente = self.partner_model.create({
+            'name': 'Intra EU supplier esente',
```

Comment:
Non so se questa entra nel .pot, ma in ogni caso per uniformità con altre stringhe dello stesso tipo presenti nel file eviterei comunque il mix dei termini. (anche in altre simili).

Reply:
:+1: Modificato